src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/DfaMatchingState.cs

```diff
@@ -29,23 +29,11 @@ internal DfaMatchingState(SymbolicRegexNode<TSet> node, uint prevCharKind)
         internal bool IsDeadend => Node.IsNothing;
 
         /// <summary>The node must be nullable here</summary>
-        internal int FixedLength
+        internal int FixedLength(uint nextCharKind)
```

Comment:
It's great that you were able to fix the fixed-length markers.  Do we know if this was contributing to some of the perf slowdowns that had been measured?

Reply:
Likely, but I don't for sure know yet. I'll measure against current main after I get this merged (to unblock Margus).